cmd/sandbox-ce/cmd/web.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/browser"
+	"github.com/pkg/errors"
+	"github.com/smacker/superset-compose/cmd/sandbox-ce/compose"
+)
+
+type webCmd struct {
+	Command `name:"web" short-description:"Web"`
+}
+
+func (c *webCmd) Execute(args []string) error {
+	if err := <-OpenUI(10 * time.Second); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func init() {
+	rootCmd.AddCommand(&webCmd{})
+}
+
+func openUI() error {
+	var stdout, stderr bytes.Buffer
+	for {
+		if err := compose.RunWithIO(context.Background(),
+			os.Stdin, &stdout, &stderr, "port", "superset", "8088"); err == nil {
+			break
+		}
+
+		time.Sleep(1 * time.Second)
+	}
+
+	address := strings.TrimSpace(stdout.String())
+	if address == "" {
+		return fmt.Errorf("no address found")
+	}
+
+	for {
+		if _, err := net.Dial("tcp", address); err == nil {
```

Comment:
this would check only that the port is open (and it is open for sure because docker-compose opened it and there is a check already above)
But it doesn't mean anything listens on it yet. You need to use http.Client or user would get "The connection was reset" or something like that.